egs/fisher_english/s5/local/semisup/chain/tuning/run_tdnn_15k_i.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+set -e
+set -o pipefail
+
+# This is fisher chain recipe for training a model on a subset of around 15 hours.
+# This is similar to _d, but uses a phone LM UNK model
+
+# configs for 'chain'
+stage=0
+tdnn_affix=7i
+train_stage=-10
+get_egs_stage=-10
+decode_iter=
+train_set=train_sup15k
+unsup_train_set=unsup_250k
+semisup_train_set=semisup15k_250k
+tree_affix=bi_i
+nnet3_affix=_semi15k_250k
+chain_affix=_semi15k_250k
+exp=exp/semisup_15k
+gmm=tri3
+xent_regularize=0.1
+hidden_dim=500
```

Comment:
Did you try to reduce it to smaller size or reducing number of layers?

Reply:
I think the Fisher dev and test are more similar to training compared to eval2000 and swbd. So it might be ok for a larger network.